### PR TITLE
zee: add depends on fmt

### PR DESCRIPTION
**Pull request by alkino**

### CODE REVIEW

**Comment by alkino** [comment]
Please retest

File: var/spack/repos/builtin/packages/zee/package.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ class Zee(CMakePackage):
     depends_on('gmsh@4: +metis~mpi+oce+openmp+shared')
     depends_on('mpi')
     depends_on('omega-h')
+    depends_on('fmt')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/zee/package.py:46]: no `when` ?

Comment (alkino) [reply var/spack/repos/builtin/packages/zee/package.py:46]: no versioning in zee.

Comment (matz-e) [reply var/spack/repos/builtin/packages/zee/package.py:46]: Awesome.

**Review by matz-e — APPROVED** [review]

**Pull request merged**